status/projects.py

```diff
@@ -746,14 +746,17 @@ def make_project_running_note(application, project, note, category, user, email)
 
         #### Check and send mail to tagged users
         pattern = re.compile("(@)([a-zA-Z0-9.-]+)")
-        userTags = pattern.findall(note)
+        userTags = [x[1] for x in pattern.findall(note)]
         if userTags:
-            RunningNotesDataHandler.notify_tagged_user(application, userTags, proj_ids, note, category, user, timestamp)
+            RunningNotesDataHandler.notify_tagged_user(application, userTags, proj_ids, note, category, user, timestamp, 'userTag')
         ####
+        ##Notify proj coordinators for all project running notes
+        proj_coord = '.'.join(doc['details']['project_coordinator'].lower().split())
```

Comment:
Might be good with a check whether project_coordinator is set here. Just to be sure.

Reply:
Added below this line.